src/contexts/RoomContext.ts

```diff
@@ -16,10 +16,15 @@ limitations under the License.
 
 import { createContext } from "react";
 
-import { IState } from "../components/structures/RoomView";
+import { IRoomState } from "../components/structures/RoomView";
 import { Layout } from "../settings/Layout";
 
-const RoomContext = createContext<IState>({
+export enum AppRenderingContext {
```

Comment:
I believe the enum name is slightly misleading here.

The values listed here are more rendering context for the message panel rather than the app.
What would you think about renaming it `MessagePanelRenderingContext` ?

Reply:
I think we have agreed on something like `TimelineRenderingType` or similar? @gsouquet

Reply:
Yes, that name sounds like a better representation of what it is